modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectLayoutLocalServiceImpl.java

```diff
@@ -389,6 +405,25 @@ private ObjectLayoutTab _addObjectLayoutTab(
 				user, objectDefinitionId,
 				objectLayoutTab.getObjectLayoutTabId(), objectLayoutBoxes));
 
+		_serviceRegistrationsMap.put(
```

Comment:
@MarinhoFeliphe hm... Instead of registering two components, we can follow the pattern we have for object definition admin screens. 

We can have a Base class for the ScreenEntry and extend It, implementing also the ScreenNavigationCategory. That way, we register just one class. And then you can override the getScreenNavigationKey and put the ObjectDefinition className + companyId + ERC like {className#companyId#ERC}

Reply:
Agreed!